packages/react-core/src/components/Select/examples/Select.md

```diff
@@ -117,7 +117,7 @@ class SingleSelectInput extends React.Component {
         <Checkbox
           label="isDisabled"
           isChecked={this.state.isDisabled}
-          onChange={this.toggleDisabled}
+          onChange={(_event, checked) => this.toggleDisabled(checked)} 
```

Comment:
on some of these files where there are a ton of `onChange`, i think i would have updated the `toggleDisabled` to have the event param to have 1 change vs a bunch.

although i'm not sure if there is a standard for whether the unused param should be here or there

Reply:
I think each `toggleDisabled` here was in a different example, so changing it at the function declaration level wouldn't have reduced the number of changes needed. Could be mistaken though.